spec/helpers/google_analytics_helper_spec.rb

```diff
@@ -1,6 +1,70 @@
 # frozen_string_literal: true
 
 describe GoogleAnalyticsHelper, type: :helper do
+
+  describe 'ga_user_properties' do
+    context 'with document' do
+      before do
+        allow(helper).to receive(:collection_show_page?).and_return(true)
```

Comment:
I'm not generally fond of stubbing a method on the thing we're testing, but given how helpers are wired up, this is probably our most direct option. Ship it.